install-paperless-ngx.sh

```diff
@@ -227,6 +227,18 @@ if [[ "$DATABASE_BACKEND" == "postgres" ]] ; then
 	POSTGRES_FOLDER=$ask_result
 fi
 
+if [[ "$DATABASE_BACKEND" == "mariadb" ]] ; then
+	echo ""
+	echo "The database folder, where mariadb stores its data."
+	echo ""
+	echo "CAUTION: If specified, you must specify an absolute path starting with /"
+	echo "or a relative path starting with ./ here."
+	echo ""
+
+	ask_docker_folder "Database folder" ""
+	MARIADB_FOLDER=$ask_result
```

Comment:
Would it make sense to combine this and the identical `POSTGRES_FOLDER` into one neutral `DATABASE_FOLDER`?

Reply:
> I'll let someone else try to run this as well. I still see that error message, but I'm willing to say it's just going to be something about my particular setup rather than an issue with this PR.

I apologize, I do see it, I wasn't checking the db container log, only the webserver log. I'm still working through trying to find where it's coming from.

Reply:
That would be ideal for sure, my concern is that it would break any kind of backwards compatibility for Postgres installs. I don't know how to publicize that kind of change in order to make the fewest people angry when their persistent storage disappears.